app/views/govuk_publishing_components/components/docs/cookie_banner.yml

```diff
@@ -18,14 +20,21 @@ examples:
       title: "Can we store analytics cookies on your device?"
       text: "This is some custom text in my cookie banner which lets users know what we're using cookies for. I can also include a link to the <a class='govuk-link' href='/cookies'>cookies page</a>"
       confirmation_message: "You’ve accepted all cookies."
+
+  with_multi_paragraph_custom_content:
+    data:
+      title: "Can we store analytics cookies on your device?"
+      text: ["This is some custom text in my cookie banner.","There are three paragraphs.","They are passed as an array"]
+      confirmation_message: "You’ve accepted all cookies."
+
   with_custom_cookie_preferences_href:
     data:
       cookie_preferences_href: "/cookies"
   in_services_asking_for_analytics_only:
     description: Use this type of banner in your service if it only uses cookies for analytics.
     data:
       title: Can we store analytics cookies on your device?
-      text: Analytics cookies help us understand how our website is being used.
+      text: "Analytics cookies help us understand how our website is being used."
```

Comment:
Why did quote marks get added to this line? Are they needed?

Reply:
Ah, added them to accommodate the <p> tags earlier, I think. Fixed.